### DISCUSSION TITLE
What's the difference between the ICU and non-ICU versions?

### DISCUSSION THREAD

**Discussion by adrian5** [userdocs/qbittorrent-nox-static#46] [Q&A] [answered]
I was wondering what its effects are, e.g. WebUI language support, Unicode filenames in torrents, etc.

**Comment by userdocs** [comment, marked as answer]
It's not a specific dependency of either libtorrent (but iconv is for v1.2) or qbittorrent. It's more about how the build process adapts automatically if ICU is present and this is mostly linked to building qtbase/qttools

Since QT and boost (but we don't build that anymore) automatically prefer ICU if it is present the main relationship would probably be between qbittorrent and QT for how Unicode characters are displayed and the webui linked into it.

To be really honest, no one seems to have a clear answer and [I've asked](https://github.com/arvidn/libtorrent/discussions/6215) on both libtorrent and qbittorrent. I have not had anyone report Unicode issues with either so all I can say it try one and if you have no issues, enjoy it.

but, ICU does seem to be a preferred library over iconv, triggered by automatic build switches, so do the builds use it just incase it turns out to be better.

**Reply by adrian5** [reply]
I see, thanks for the insight. And thanks a lot for maintaining this repo, just recently discovered it.
